packages/console/src/hooks/use-tenant-pathname.ts

```diff
@@ -65,8 +65,20 @@ function useTenantPathname(): TenantPathname {
   const href = useHref('/');
 
   const match = useCallback(
-    (pathname: string, exact = false) =>
-      matchPath(joinPath(':tenantId', pathname, exact ? '' : '*'), location.pathname) !== null,
+    (pathname: string, exact = false) => {
+      // Match relative pathnames directly
+      if (pathname.startsWith('.')) {
+        return (
+          matchPath(joinPath(location.pathname, pathname, exact ? '' : '*'), location.pathname) !==
+          null
+        );
+      }
+
+      // Match absolute pathnames with the tenant segment
+      return (
+        matchPath(joinPath(':tenantId', pathname, exact ? '' : '*'), location.pathname) !== null
```

Comment:
How about a path like `application/:id`, which does not start with `.` but I think should also be treated as relative path.
Ideally only path starts with backslash `/` should be treated as absolute path, right?

Reply:
yes i can update to this